### PR TITLE
Test serve start config

### PR DESCRIPTION
**Pull request by gsnedders**
Mostly throwing this up because I think this is a terrible test but I don't know how to do better? And it shows a real issue.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/9726)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2018-03-01 02:31:22
Finished: 2018-03-01 02:43:19


<h2>Failing Jobs</h2>
<ul>

<li>tools_unittest in py27</li>

<li>tools_unittest in pypy</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25067)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/347578438)

File: tools/wptrunner/wptrunner/tests/test_products.py [hunk 1]
```diff
@@ -3,12 +3,23 @@
 
 from os.path import join, dirname
 
+import mock
 import pytest
 
-sys.path.insert(0, join(dirname(__file__), "..", ".."))
+sys.path.insert(0, join(dirname(__file__), "..", "..", "..", "..")) # /
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/tests/test_products.py]: Why do you need both of these on the path?

**Review by jgraham — COMMENTED** [review]
I don't love this either, but I guess it's OK.

File: tools/wptrunner/wptrunner/tests/test_products.py (hunk 1 above)
Comment (gsnedders) [reply tools/wptrunner/wptrunner/tests/test_products.py]: Um, because I didn't change everything to use `from tools`, seemingly. Fixed.

**Comment by gsnedders** [comment]
Note that this covers both #9750 and #9651.

**Comment by gsnedders** [comment]
@jgraham if it's okay can we land it?

File: tools/wptrunner/wptrunner/browsers/chrome_android.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def env_extras(**kwargs):
 
 
 def env_options():
-    return {"bind_hostname": "true"}
+    return {"bind_hostname": True}
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/browsers/chrome_android.py]: Let's just make `True` the default and remove all of these

Comment (gsnedders) [reply tools/wptrunner/wptrunner/browsers/chrome_android.py]: done

**Review by jgraham — APPROVED** [review]

**Pull request merged**